packages/rollup-core/src/app/l2-node-service.ts

```diff
@@ -23,15 +23,15 @@ export class BlockBatchSubmitter implements BlockBatchListener {
   /**
    * @inheritDoc
    */
-  public async handleBlockBatches(blockBatches: BlockBatches): Promise<void> {
+  public async sendBlockBatches(blockBatches: BlockBatches): Promise<void> {
```

Comment:
Yeah, I think a rename is in order since we will no longer be sending all batches for an L1 block together, and each one will be a single batch. This requires a change on the go side as well, so just left it for now.

Reply:
Created a ticket https://optimists.atlassian.net/browse/YAS-499